tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -2537,9 +2537,7 @@ def binary_op(
     return binary_op
 
 
-eltwise_add_optional = make_binary_op_optional_output(ttl.tensor.add)
```

Comment:
Why are add and mul removed?

Reply:
I have created a separate test for them just below to pass `queue_id`. for now, only eq, add and mul have queue_id support. in future if we are to add `queue_id` for all the binary ops, we'll use the `make_binary_op_optional_output` to test all at once.